content/admin/onboard_team/CreateUsers.md

```diff
@@ -4,41 +4,106 @@ description: "Creating Users in Contrast TeamServer"
+Enterprise-on-Premises (EOP) customers can delegate users to perform system administration functions across organizations such as managing users, groups, applications, licenses, API keys and security policies. This assumes that you created multiple organizations in Contrast as part of a multi-tenant deployment. See [Granting and Revoking SuperAdmin Permissions](admin-manageorgs.html#sa) to get started.
 
-You may decide to designate an [Application Access Group](admin-onboardteam.html#group) which can provide more administrative function for an application or restrict what applications the user can view or modify.
+To create users as a System Administrator, go to the **User menu** and choose **SuperAdmin** in the **Use Contrast Security as** section. Select the **Users** page in the top navigation. 
 
-Enterprise On-Premises (EOP) customers have the ability to delegate users to perform system administration functions across organizations such as managing users, groups, applications, licenses, API keys and security policies. This assumes multiple organizations have been created within Contrast as part of a multi-tenant deployment. See [Granting and Revoking SuperAdmin Permissions](admin-manageorgs.html#sa) to get started.
+### Individual users
 
-## Creating Users as an Organization Administrator
+To add a single user, click the button to **Add User** above the grid, and complete the following steps: 
 
-1. Log in to Contrast
-2. Navigate to the user menu in the upper right 
-3. Select **Organization Settings**
-4. Select **Users** in the left navigation
-5. Click the **Add User** button
-6. Provide the required inputs (Email, First & Last Name, and default Organization Role)
+* Enter the user's **First Name**, **Last Name** and **Email Address** in the provided fields. 
+* Check the box if you want to **Require email activation** instead of requiring a password.
+* Choose which of the **System Roles** shoud apply to the user in the dropdown menu. The default is **None**. 
+* Choose the **Organization** to which the user belongs. 
+* Once you decide on an organization, you can choose the default **Organization Role** in the dropdown menu as well as an **Application Access Group** in the multiselect field.
+* Choose the **Date Format**, **Time Format** and **Time Zone** in the dropdown fields. 
+* The box to **Use Organization Settings** is checked by default. You can uncheck the box to create your own settings using the **Access** toggle or the box to **make user API only**. 
+* Click the **Add** button to save the user's information. 
 
->**Note:** Make sure you verify the [Role](admin-manageorgsroleperm.html#roles) so that users will have the privileges to carry out the actions you intend.
+<a href="assets/images/User-system-admin.png" rel="lightbox" title="User page"><img class="thumbnail" src="assets/images/User-system-admin.png"/></a>
```

Comment:
Can we make the image captions a little more descriptive and consistent with capitalization? Maybe "SuperAdmin - Add User" or "Adding a user as a SuperAdmin" ?